content/announcements/2025-03-24-kubecon-eu-announcments.md

```diff
@@ -0,0 +1,43 @@
+---
+slug: cert-manager-kubecon-eu-2025
+title: Come see cert-managers at KubeCon EU 2025 in London!
+description: Here's all the details you need for cert-manager related events at KubecCon EU 2025.
+date: "2025-03-24T12:00:00Z"
+---
+
+KubeCon EU 2025 is right around the corner and if you're lucky enough to be attending in London, we would love to see you!
+Below are all the details of the events we have in London so you can bookmark them all.
```

Comment:
nitpick (non-blocking): Minor style suggestion (wouldn't have added this if there wasn't a separate blocker, but feel free to ignore!)

```suggestion
KubeCon EU 2025 is right around the corner and if you're lucky enough to be attending in London, we would love to see you!

Below are all the details of the events we'll be at so you can bookmark them all and join in the fun!
```